tests/list_of_tests.cmake

```diff
@@ -1962,12 +1962,12 @@ four_c_test_nested_parallelism(sohex8_multiscale_macro_2micro.dat sohex8_multisc
 four_c_test_nested_parallelism(tsi_heatconvection_monolithic.dat tsi_heatconvection_monolithic.dat "")
 
 # testing the whole framework: from pre_exodus and 4C to the post-filter:
-four_c_test_framework(tutorial_battery 2 "")
-four_c_test_framework(tutorial_contact_3d 2 "")
-four_c_test_framework(tutorial_fluid 2 "")
-four_c_test_framework(tutorial_fsi 2 "")
-four_c_test_framework(tutorial_fsi_3d 2 fsi_part_struct_solver.xml)
-four_c_test_framework(tutorial_solid 3 xml/multigrid/elasticity_template.xml)
+four_c_test_framework(PREFIX tutorial_battery NP 2)
+four_c_test_framework(PREFIX tutorial_contact_3d NP 2)
+four_c_test_framework(PREFIX tutorial_fluid NP 2)
+four_c_test_framework(PREFIX tutorial_fsi NP 2)
+four_c_test_framework(PREFIX tutorial_fsi_3d NP 2 COPY_FILES ${PROJECT_SOURCE_DIR}/tests/input_files/fsi_part_struct_solver.xml)
+four_c_test_framework(PREFIX tutorial_solid NP 3 COPY_FILES ${PROJECT_SOURCE_DIR}/tests/input_files/xml/multigrid/elasticity_template.xml)
```

Comment:
@dharinib98 @mayrmt You would append your new file from #400 in the back. COPY_FILES takes an arbitrary number of arguments.

Reply:
@sebproell Great! Thanks a lot!